0.35.2 Release Notes (#3986)
astronomer/docs: software/runtime-version-lifecycle-policy.mdx

Maintenance is discontinued the last day of the month for a given version. For example, if the maintenance window for a version of Astro Runtime is January - June of a given year, that version will be maintained by Astronomer until the last day of June.

### Restricted runtime versions

> There are some restricted versions of the Astro runtime that have known bugs and can't be used even if you enabled [deprecated Astro Runtime versions](https://docs.astronomer.io/astro/upgrade-runtime#run-a-deprecated-astro-runtime-version).

Author: ```suggestion
There are some restricted versions of the Astro runtime that have known bugs and shouldn't be used.
```
